Kinvey/Kinvey/Error.swift

```diff
@@ -89,6 +89,14 @@ public enum Error: Swift.Error, CustomStringConvertible, CustomDebugStringConver
         }
     }
     
+    public var errorDescription: String? {
+        return description
+    }
+    
+    public var failureReason: String? {
+        return description
```

Comment:
not really, but if someone's code is relying in one of them it could be useful. Also, strictly speaking, `errorDescription` and `failureReason` have similar meaning. One thing that we can do in the future is use the `errorDescription` as overview of the error and leave the `failureReason` to describe in more details the reason why error happened. Any thoughts? @tejasranade what's your opinion?